Doc/library/dataclasses.rst

```diff
@@ -0,0 +1,588 @@
+:mod:`dataclasses` --- Dataclasses
+==========================================
+
+.. module:: dataclasses
+    :synopsis: Container datatypes
```

Comment:
Is the synopsis here correct? (I think I copy pasted from something else...)

Reply:
Fixed. Good catch!